sysdig-probe-loader

```
@@ -764,7 +764,7 @@ if ([ "$SYSDIG_AGENT_DRIVER" = universal_ebpf ] || [ "$SYSDIG_AGENT_DRIVER" = km
   exit 1
 elif [ "$SYSDIG_AGENT_DRIVER" = universal_ebpf ]; then
   # We don't need the kmod for Universal eBPF
-  /sbin/rmmod "$PROBE_NAME"
+  /sbin/rmmod "$PROBE_NAME" || true
```

Should we check if it's there, instead of just ignoring the failure?
```suggestion
  lsmod | grep -q "^$PROBE_NAME " && /sbin/rmmod "$PROBE_NAME"
```

---

I would feel better about replacing this line with `remove_kernel_probe || or true` instead.  (See `remove_kernel_probe` function source [here](https://github.com/draios/probe-builder/blob/31f6ec4d72a46b3f1039f88df85b9fb9a12705cf/sysdig-probe-loader#L189))

It already implements @deepskyblue86's idea of checking for the presence of the kernel module before trying to `rmmod` it.

Also, I don't have strong feelings about whether we use `|| true` to ignore an `rmmod`/`remove_kernel_probe` failure, or not.  I will defer to your judgement there.